### PR TITLE
Cilium Charts set the persistent keepalive for cilium_wg0

### PR DESCRIPTION
**Pull request by chaunceyjiang**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)
- [x] Thanks for contributing!

Fixes: #https://github.com/cilium/cilium/pull/27932#pullrequestreview-1610630959

```release-note
Cilium Charts set the persistent keepalive for cilium_wg0
```

### CODE REVIEW

**Comment by chaunceyjiang** [comment]
CI failed, the reason is the following content.
```
Please fix the following spelling mistakes:
* Documentation/helm-values.rst:960: (Wireguard)  Controls Wireguard PersistentKeepalive option. Set 0 to disable.
* Documentation/helm-values.rst:960: (Wireguard)  Controls Wireguard PersistentKeepalive option. Set 0 to disable.
```

So, How can i fix these mistakes? @youhonglian Can you help me ?

**Comment by nebril** [comment]
@chaunceyjiang `WireGuard` needs to be added to Documentation/spelling_wordlist.txt

**Review by youngnick — APPROVED** [review]

**Review by nebril — APPROVED** [review]
Thanks!

**Comment by chaunceyjiang** [comment]
> @chaunceyjiang WireGuard needs to be added to Documentation/spelling_wordlist.txt

@nebril Thanks~  I ran it again. `Documentation/update-spelling_wordlist.sh Wireguard`

Can you help me run e2e tests again?

**Comment by chaunceyjiang** [comment]
Hi @learnitall Please tale a look. Thanks~

**Review by fujitatomoya — APPROVED** [review]
lgtm

File: Documentation/helm-values.rst [hunk 1]
```diff
@@ -956,6 +956,10 @@
      - Encryption method. Can be either ipsec or wireguard.
      - string
      - ``"ipsec"``
+   * - :spelling:ignore:`encryption.wireguard.persistentKeepalive`
+     - Controls Wireguard PersistentKeepalive option. Set 0 to disable.
```
Comment (learnitall) [inline Documentation/helm-values.rst]: I think there might be a typo here. Do users need to set to `0` or `0s` to disable?

```suggestion
     - Controls Wireguard PersistentKeepalive option. Set to 0s to disable.
```

**Comment by learnitall** [comment]
Hey @chaunceyjiang thank you for your patience. One quick comment here.

**Pull request merged**